R/TaskClassifST.R

```diff
@@ -130,6 +122,8 @@ TaskClassifST = R6::R6Class("TaskClassifST",
   catf(str_indent("Properties:", self$properties))
   catf(str_indent("Coordinates", self$coordinate_names))
 
+  # TODO: is this copy-pasted from Task?
+  # We should try to find a better way.
```

Comment:
Yes. With the aim of also printing the coords.

Maybe we can add an argument to `mlr3::task_print()` accepting a list of "type:column", i.e. `list(Coordinates = coordinate_names)` for internal extension capabilities?

Reply:
Yep, we maybe need to formalize the printing stuff more and implement a two-step approach: one function which converts into such a named list, and a second one for printing.